src/api/middleware/validators/auth.js

```diff
@@ -3,19 +3,19 @@ const { body } = require("express-validator");
 const { useExpressValidators } = require("../errorHandler");
 const Account = require("../../../models/Account");
 
-const checkDuplicateUsername = async (username) => {
-    const acc = await Account.findOne({ username }).exec();
+const checkDuplicatedEmail = async (email) => {
+    const acc = await Account.findOne({ email }).exec();
     if (acc) {
-        throw new Error("Username already exists");
+        throw new Error("Email already exists");
```

Comment:
I'm not entirely sure what message is being sent in other custom validators, but we should probably also use the validator reasons enum here (and elsewhere).